example/lambdas/python-processing/requirements.txt

```diff
@@ -1,4 +1,4 @@
-boto3==1.12.16
+boto3~=1.12.47
```

Comment:
Do we still need this `requirements.txt` file if we have Pipfile and Pipfile.lock?

Reply:
Hmmm, I guess not. I can delete them before final merge, but until the build passes and I'm 100% certain the Pipfile is correct, I plan to hang on to them.

Reply:
https://github.com/nasa/cumulus/pull/2038/commits/a167671b8514c0994b45218b1acc767bae029fe0